server/src/instant/model/schema.clj

```diff
@@ -4,13 +4,22 @@
             [instant.jdbc.aurora :as aurora]
             [instant.db.datalog :as d]
             [instant.model.rule :as rule-model]
-            [instant.db.permissioned-transaction :as permissioned-tx])
+            [instant.db.permissioned-transaction :as permissioned-tx]
+            [instant.util.exception :as ex])
   (:import (java.util UUID)))
 
 (defn map-map [f m]
   (into {} (map (fn [[k v]] [k (f [k v])]) m)))
 
-(defn schemas->ops [current-schema new-schema]
+(defn link-dir-keys [{ri :reverse-identity
+                      fi :forward-identity}]
+  (concat [[(-> fi second)
```

Comment:
I would write: 

`(defn attr-ident-names [attr] 
     (keep identity [(attr-model/fwd-ident-name attr) (attr-model/rev-ident-name attr)))`

Reply:
`keep identity` neat

Reply:
`fwd-ident-name` returns an empty list in the null case, so it was `keep seq`